lib/broccoli/angular-broccoli-bundle.js

```diff
@@ -16,14 +16,18 @@ class BundlePlugin extends Plugin {
   build() {
     var relativeRoot = path.relative(process.cwd(), this.inputPaths[0]);
     var builder = new Builder(relativeRoot, `${relativeRoot}/system-config.js`);
-    return builder.bundle('main - [app/**/*]',
+    const bundles = this.options.bundles || ['app'];
+
+    return builder.bundle('main' + bundles.map(name => `[${name}/**/*]`).join(' - '),
```

Comment:
For `bundles = ['app1', 'app2']` this will evaluate to `[app1/**/*] - [app2/**/*]`. I think it's missing the first `-`.